web/src/containers/Login.js

```diff
@@ -36,50 +37,52 @@ class Login extends Component {
 
     return (
       <Fragment>
-        <header className="clearfix px2 py1 bg-white">
-          <div className="left">
-            <Link to="/" className="btn px0 bold caps">
-              {t('titleBasic')}
-            </Link>
-          </div>
-        </header>
-
-        <div className="mx-auto my3 p2 sm-col-6 md-col-4 bg-white rounded">
-          <h1 className="mt0 h2">Please log in.</h1>
-          {error && (
-            <div className="mb2 p1 h6 alert alert-error">
-              <strong>Sorry!</strong> Something went wrong. Please try again.
-            </div>
-          )}
-          <form onSubmit={this.handleSubmit}>
-            <div className="mb2">
-              <label htmlFor="username">Email</label>
-              <input
-                id="username"
-                type="text"
-                name="username"
-                className="input"
-                value={username}
-                onChange={this.handleChange}
-              />
+        <Header />
+        <div className="card--container">
+          <div className="ds-l-container card">
+            <div className="ds-l-row">
+              <div className="ds-l-col--1 ds-u-margin-left--auto" />
+              <div className="ds-l-col--12 ds-l-sm-col--10 ds-l-lg-col--6">
+                <h1 className="ds-h1">Log in</h1>
+                {error && error==="Unauthorized" && (
```

Comment:
This feels a little fragile to me somehow, but I wasn't sure how else to approach it.

Reply:
In #1399 I did this by having the redux action convert the error message into a human string:

https://github.com/18F/cms-hitech-apd/pull/1399/files#diff-1af493675e4b7123516353a186f85abeR79

Then I have it reject the promise with that message.  Back in the UI components, I have them `await`/`catch` the action so they'll know when it completes or fails:

https://github.com/18F/cms-hitech-apd/pull/1399/files#diff-c6793634e433d7ad10c9c8ea93ee8005R52

But I'm not 100% in love with this pattern because it's hooking the UI directly to the actions instead of going through the state, which I think is only a problem because it's inconsistent.  However, I wasn't sure how else to handle it.  Maybe we use one big `error` state component that tracks every error message.  🤔  Not a bad approach, I think.  Maybe I'll try updating #1399 after this one is merged.

All that said...  this seems fine for now.  😛